plSqlStandards.md

```diff
@@ -0,0 +1,316 @@
+﻿# PL/SQL Coding Standards
+
+## Rationale
+
+This document is intended to present the standards to which all PL/SQL
+code produced by, or on behalf of, DEFRA should adhere. PL/SQL coding
+standards are intended to:
+
+  - Make the produced code more easily read and understood.
+
+  - Make the code more easily maintainable.
+
+  - Make the code more reliable.
+
+## The Derivation of these Standards
+
+These standards have been derived from hands-on experience at the
+Environment Agency and Rural Payments Agency over many years. As such
+they are mature, comprehensive while also being reasonably lightweight
+and follow standard industry practice.
+
+## Important Recommendation: Refer to these Standards by Version as well as Name.
```

Comment:
Rather than having this here, I think it should be covered on the standards front page as it applies to all the standards.

I'd welcome some input from @Cruikshanks

Reply:
I agree, if this what we're saying when it comes to standards.

So as a first pass should we be saying this document should have a version number (in the title, as a section heading ??) Then this section about versioning be moved to `standards/README.md`.